### ISSUE TITLE
Merge conflict diff not shown in VSCode

### ISSUE THREAD

**Issue by minseoksuh** [microsoft/vscode#130179]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: Version: 1.58.2 (Universal) & Version: 1.59.0-insider (Universal)
- OS Version: MacOs Big Sur 11.4 (20F71)

Steps to Reproduce:

1.  clone [https://github.com/minseoksuh/super-fun-time](https://github.com/minseoksuh/super-fun-time)
2. pull branch "conflict-test"
3. Merge Conflict will occur, but the conflict diff will not show. And the code you are trying to merge(from "conflict-test" is gone)
4. 
![Screen Shot 2021-08-05 at 8 51 47 PM](https://user-images.githubusercontent.com/54365519/128345450-f4539fa6-735c-4707-9b4c-3a3a0ba56e65.png)

- CodeLens is enabled
- Have uninstalled and reinstalled vscode
- conflict diff seems to be working fine in other projects

Maybe I am doing something wrong, but I just cant find any solutions myself.
Please help.

Thank you in advance!

**Comment by minseoksuh** [comment]
Just checked #114397
But downgrading to version 1.51.1 does not seem to work for me.

**Comment by minseoksuh** [comment]
I figured out what my issue was.

The problem was that the encoding of my README.md file was in UTF-16LE.
[https://stackoverflow.com/questions/19722971/set-file-as-non-binary-in-git](https://stackoverflow.com/questions/19722971/set-file-as-non-binary-in-git)

so I created a new README2.md in both branches using
`iconv -f UTF-16LE -t utf-8 README.md`
and then tried to merge

![Screen Shot 2021-08-06 at 12 07 35 AM](https://user-images.githubusercontent.com/54365519/128373935-8cf3d3c7-d29a-456a-8309-50fdaea110ce.png)

the conflict between two README2.md was caught correctly.

Sorry for raising an issue that is not related to VSCode.
Maybe someone can get help from this issue.

Thanks anyways!
